backbone.layoutmanager.js

```diff
@@ -731,7 +741,9 @@ var LayoutManager = Backbone.View.extend({
       if (view.hasRendered || force) {
         LayoutManager._removeView(view, force);
       }
-    });
+
+    // call value() in case this chain is evaluated lazily
+    }).value();
```

Comment:
Same issue here, but we can get away with calling `.value()` to execute the chain because we're not returning anything.